Game/level.gd

```diff
@@ -98,3 +100,14 @@ func _goal_reached(player: Node2D):
 	
 func _on_audio_stream_player_finished():
 	$AudioStreamPlayer.play()
+
+func _on_restart_button_pressed():
+	$VictoryScreen/Victory.hide()
+	$Goal.hide()
+	
+	loading_indicator.hide()
```

Comment:
I'm pretty sure this is already hidden.

Reply:
i did it for safe measure, it didnt hide it before i added it so, decided to take precaution

Reply:
Fair enough.